SIL.ReleaseTasks.Dogfood/README.md

```diff
@@ -3,7 +3,9 @@
 This is basically the same as SIL.ReleaseTasks with the exception that it only contains the
 `SetReleaseNotesProperty` task. The project creates a nuget package that then can be included
 in `SIL.ReleaseTasks`. This nuget package isn't intended to be used in general - use
-`SIL.ReleaseTasks` instead. It won't be uploaded to nuget.org regularly. The only reason it
-exists is to be able to use the `SetReleaseNotesProperty` task in `SIL.ReleaseTasks`
+`SIL.ReleaseTasks` instead. It won't be uploaded to nuget.org. Instead, it uses a local package
+source. The package does not get built as part of the project build inside VS, so to rebuild it,
+run msbuild /t:Restore from the build folder.
```

Comment:
Running `msbuild /t:Restore` works but looks a little odd. Running `msbuild /t:PackDogfood` would also work - which is what `Restore` depends on...